.eslintrc.json

```diff
@@ -1,5 +1,7 @@
 {
-  "extends": ["eslint:recommended"],
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint", "libram"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
   "rules": {
     "block-scoped-var": "error",
     "eqeqeq": "error",
```

Comment:
I realize this is all my code, but can we sort this block (everything before sort-imports) alphabetically?